src/style.css

```diff
@@ -1 +1,90 @@
 /* Styles go here */
+
+html {
+  font-family: "Roboto", sans-serif;
+  --color: #00acdc;
```

Comment:
add css variables to `:root` selector

Reply:
It means you can add separate `:root` selector just for variables and leave `html` as before

Reply:
Mentors meant that in this selector you only need to add variables and leave the font in `html` selector, as you had before